tsconfig.json

```diff
@@ -7,6 +7,7 @@
     "types": ["node", "webdriverio/async"],
     "declaration": true,
     "strict": true,
+    "skipLibCheck": true,
```

Comment:
Same as here: https://github.com/cloudscape-design/demos/pull/151#discussion_r1716904318

There is a compatibility issue between certain versions of webdriverio and typescript. Still, it is better to have aligned versions across packages so we can trust our tests more

Reply:
Not sure what "consumers" you mean here. We only use this package inside our own repos and for `@cloudscape-design/components`, for example it is a dev-only dependency.

Reply:
Moreover, this PR only changes package-lock.json, no consumers should be affected

Reply:
Dev dependencies do not matter, what do I miss here?